fireplace/cards/wog/druid.py

```diff
@@ -11,3 +11,14 @@ class OG_313:
 	)
 
 OG_313e = buff(+1, +1)
+
+
+class OG_202:
+	"Mire Keeper"
+	choose = ("OG_202a", "OG_202b")
+
+class OG_202a:
+	play = Summon(CONTROLLER, "OG_202c")
+
+class OG_202b:
```

Comment:
Oh also, these all should go *after* Addled Grizzly (the cards are ordered by id)